pkg/frontend/validate.go

```diff
@@ -77,7 +77,7 @@ func (f *frontend) validateOpenShiftUniqueKey(ctx context.Context, doc *api.Open
 var rxKubernetesString = regexp.MustCompile(`(?i)^[-a-z0-9.]{0,255}$`)
 
 func validateAdminKubernetesObjectsNonCustomer(method, groupKind, namespace, name string) error {
-	if !utilnamespace.IsOpenShift(namespace) {
+	if !utilnamespace.IsOpenShiftNamespace(namespace) {
```

Comment:
It'd be great to have some consistency on how we import that, why do we import it differently here than from the other files ? Also, maybe I'd call it `namespaceutil` if we are going to import it with that kind of name (I remember some packages in the std library that were named this way.  (e.g. `ioutil` even though it has been deprecated). 
Other solution is to give a less generic name to the package so we don't have conflicts on package names and don't have to rename it. It's up to you 

Apart from that little nit, LGTM

Reply:
Hi @facchettos - Thanks for your feedback. I think the reason we imported namespace with a different name (`utilnamespace`) is to distinguish with the `namespace` parameter in the function `func validateAdminKubernetesObjectsNonCustomer(method, groupKind, namespace, name string) error`. I am happy to update to use `namespaceutil` if it's a standard naming convention.